buildconfig/config_emsdk.py

```diff
@@ -224,6 +266,8 @@ def main(auto_config=False):
     return DEPS
 
 
-if __name__ == '__main__':
-    print("""This is the configuration subscript for Unix.
-Please run "config.py" for full configuration.""")
+if __name__ == "__main__":
+    print(
+        """This is the configuration subscript for Unix.
+Please run "config.py" for full configuration."""
+    )
```

Comment:
to simplify diffs on this file, I'd prefer if black formatting is not applied to this file for now

Reply:
changed the whole message.